File: Dermatoloska_ordinacija/UI/dermatoloska_mobile/lib/models/korisnik.dart
import 'package:dermatoloska_mobile/models/tipKorisnika.dart';
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? korisnikId;
  String? ime;
  String? prezime;
  String? username;
  String? email;
  String? telefon;
  String? adresa;
  String? slika;
  TipKorisnika? tipKorisnika;

  Korisnik(this.korisnikId, this.ime, this.prezime, this.username, this.email, this.telefon, this.adresa, this.slika, this.tipKorisnika);

  factory Korisnik.fromJson(Map<String, dynamic> json) => _$KorisnikFromJson(json);

  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}

File: Dermatoloska_ordinacija/DermatoloskaOrdinacija.Model/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DermatoloskaOrdinacija.Model
{
    public partial class Korisnik
    {
        public int KorisnikId { get; set; }

        public string? Ime { get; set; }

        public string? Prezime { get; set; }

        public string? Username { get; set; }

        public string? Email { get; set; }

        public string? Telefon { get; set; }

        public string? Adresa { get; set; }
        public byte[]? Slika { get; set; }

        public int? SpolId { get; set; }


        public TipKorisnika? TipKorisnika { get; set; } 
        public int? TipKorisnikaId { get; set; }
    }
}
